core/jobs/batch_jobs/exp_migration_jobs.py

```diff
@@ -96,8 +98,8 @@ def _generate_exploration_changes(
         exp_id: str, exp_model: exp_models.ExplorationModel
     ) -> Iterable[Tuple[str, exp_domain.ExplorationChange]]:
         """Generates exploration change objects. ExplorationChange object is
-        generated schema version for some field is lower than the latest
-        schema version.
+        only generated when schema version is lower than the latest schema
```

Comment:
The ExplorationChange object is only generated when the exploration's states schema version is lower than ...

(assuming this is what you mean? I think with schema versions it's important to be very clear what you're talking about)

Reply:
Done.